docs/REST API/rest-api.md

```diff
@@ -7373,22 +7373,20 @@ This allows you to easily configure scheduled queries that will impact a whole t
 
 - [Run script](#run-script)
 - [Get script result](#get-script-result)
-- [Run live script](#run-live-script)
 - [Add script](#add-script)
 - [Delete script](#delete-script)
 - [List scripts](#list-scripts)
 - [Get or download script](#get-or-download-script)
 - [Get script details by host](#get-script-details-by-host)
 
-### Run script
+### Run scripts
 
-Run a script on a host.
+In Fleet there are 2 types of scripts:
 
-The script will be added to the host's list of upcoming activities.
+- Fleet UI Scripts - script files that have been uploaded.
+- "Live" Scripts - script files that have been uploaded, or, a script body included with a fleetctl command on the command line or in an API request to execute a script.
 
-The new script will run after other activities finish. Failure of one activity won't cancel other activities.
-
-`POST /api/v1/fleet/scripts/run`
+When a Fleet UI script is executed it will be added to the Host Activity queue. The script will run after other activities finish. Failure of one activity won't cancel other activities.
```

Comment:
I think scripts run via the API will be added to the activity queue as well, not just via the UI; can just say "When a script is executed"

(Also no capitalization for "host activity" — and let's call this "host's upcoming activity" so it matches the UI naming)